diagnostic.md

```diff
@@ -7,40 +7,54 @@ What command do we use to do that? Use the **correct** URL from your fork of
 this repository in your answer.
 
 ```sh
-<!-- Remove this comment and place your answer here. -->
+git clone git@github.com:joebelmonte/git-diagnostic.git
 ```
 
 2.What do you do after cloning a repository, before you start making any
 changes/additions?
 
-<!-- Remove this comment and place your answer here. -->
+git init
```

Comment:
If you clone down a repository, it will already be a git repository and you will not need to use `git init`. In this case, we would simply want to create a new branch